src/utils/sendEmail.ts

```diff
@@ -16,13 +16,6 @@ export const sendEmail = () => {
     text: "and easy to do anywhere, even with Node.js",
     html: "<strong>and easy to do anywhere, even with Node.js</strong>",
   };
-
-  sgMail
-    .send(msg)
-    .then(() => {
-      console.log("Email sent");
-    })
-    .catch((error) => {
-      console.error(error);
-    });
+  // eslint-disable-next-line @typescript-eslint/no-floating-promises
+  sgMail.send(msg)
```

Comment:
failed email send is unhandled? 

I guess you haven't really built out execution logs for these tasks so its OK for now

Reply:
> failed email send is unhandled?
> 
> I guess you haven't really built out execution logs for these tasks so its OK for now

Yeah execution logs aren't built out and showing the error in the UI is obsolete as the user won't be on the site (most probably) when the webhook is executed. Having this unhandled is probably the only option for now.